docs/APP.md

```diff
@@ -1,49 +1,42 @@
-# Apps API Reference (`@aragon/client`)
+## Install
```

Comment:
Is this the right place for installation instructions? It seems unexpected to have these above the title of the page.